mandatory/3-function-output.js

```diff
@@ -9,6 +9,7 @@ function combine2Words(word1, word2) {
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
+  return firstWord + ' ' + secondWord + ' ' + thirdWord
```

Comment:
That's right, you can concatenate multiple String variables simply by how you did.